test/Chirp.Razor.UnitTest/ChirpStorageTests.cs

```diff
@@ -0,0 +1,141 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Xunit;
+using Chirp.Razor.Shared.Storage;
+using Microsoft.Data.Sqlite;
+
```

Comment:
There is no namespace declared
```suggestion
namespace Chirp.Razor.UnitTest;
```